Week4/git-check.py

```diff
@@ -0,0 +1,39 @@
+from cgitb import text
+import re
+import subprocess
+
+def git_blam(line_range, f):
+    #Restrict the output within certain line range
+    subprocess.run(["git","blame", "-L", line_range, f, ">>", "text.txt"])
+
+def git_Ignore():
+    #Ignore whitespace changes
+    subprocess.run(["git","blame", "-w",  "/merge.txt"])
+
+def git_log():
+    #Track file content changes
+    subprocess.run(["git", "log", "-S", "Bob", "--/merge.txt"])
+
+if __name__ == "__main__":  
+    f1 = "merge.txt"
+    rang = ['6,8', '15,17', '24,29']
```

Comment:
the requirement is to find out the range programmatically rather than hard coding